### DISCUSSION TITLE
Create Multiple Routes with Solito

### DISCUSSION THREAD

**Discussion by Kemuel-Batista** [nandorojo/solito#335] [Q&A]
Hi Guys, I´m discovering many things with solito project but i´m using now with a personal project but I have a doubt with routes because I created a template with tailwind and I need to create two routes (one with unauthorized routes with some pages and a authorized page, I´m using the next project to build my app, if anyone can help me

**Comment by JWesorick** [comment]
https://nextjs.org/docs/routing/introduction
